### PR TITLE
Update service configs to include locales

### LINKED ISSUE #3819: Update service configs to include locales.
**Is your feature request related to a problem? Please describe.**
We are not including the relevant `locales` in our config, this leads to some services displaying timestamps in english.

**Describe the solution you'd like**
Include relevant locales to their respective services. For some services we are overriding the default moment locales. For those services include locale from `psammead-locales`.
More details [here](https://bbc.github.io/psammead/?path=/info/utilities-psammead-locales--moment-arabic-ar)

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by AlistairGempf**
Resolves #3819

**Overall change:** Use the updated psammead-locales in the relevant services. Services updated are:
- Afaan Oromoo
- Amharic
- Azeri
- Bengali
- Gahuza
- Gujarati
- Hausa
- Hindi
- Marathi
- Mundo
- Nepali
- Portuguese
- Russian
- Serbian (Latin and cyrillic)
- Sinhala
- Somali
- Swahili
- Tamil
- Thai
- Tigrinya
- Ukrainian
- Urdu
- Uzbek

**Code changes:**

- Include the updated psammead-locales into the relevant service configs.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by DenisHdz — DISMISSED** [review]
Looks good 🚀

**Comment by DenisHdz** [comment]
Can you wait until [this](https://github.com/bbc/psammead/pull/2227) PR is merged so you can also update those services?

File: src/app/lib/config/services/serbian.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import {
 } from '@bbc/gel-foundations/scripts';
 import { serbian as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/GMT';
+import '@bbc/psammead-locales/moment/sr-cyrl';
+import '@bbc/psammead-locales/moment/sr';
```
Comment (dr3) [inline src/app/lib/config/services/serbian.js:9]: The locales used (at the bottom of this file) are `sr-latn` (not `sr`) and `sr-cyrl`

File: src/app/lib/config/services/afaanoromoo.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { cyrillicAndLatin } from '@bbc/gel-foundations/scripts';
 import { afaanoromoo as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/Africa/Addis_Ababa';
+import '@bbc/psammead-locales/moment/om';
```
Comment (dr3) [inline src/app/lib/config/services/afaanoromoo.js:5]: `datetimeLocale: 'om-et'` needs to be `datetimeLocale: 'om'`

File: src/app/lib/config/services/amharic.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ import {
   F_NOTO_SANS_ETHIOPIC_REGULAR,
 } from '@bbc/psammead-styles/fonts';
 import '@bbc/moment-timezone-include/tz/Africa/Addis_Ababa';
+import '@bbc/psammead-locales/moment/am';
```
Comment (dr3) [inline src/app/lib/config/services/amharic.js:9]: `datetimeLocale: 'am-et'` needs to be `datetimeLocale: 'am'`

File: src/app/lib/config/services/gahuza.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { cyrillicAndLatin } from '@bbc/gel-foundations/scripts';
 import { gahuza as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/GMT';
+import '@bbc/psammead-locales/moment/rw';
```
Comment (dr3) [inline src/app/lib/config/services/gahuza.js:5]: `datetimeLocale: 'rw-rwt'` needs to be `datetimeLocale: 'rw'`

File: src/app/lib/config/services/gujarati.js [hunk 5]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { devanagariAndGurmukhi } from '@bbc/gel-foundations/scripts';
 import { gujarati as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/Asia/Kolkata';
+import '@bbc/psammead-locales/moment/gu';
```
Comment (dr3) [inline src/app/lib/config/services/gujarati.js:5]: `datetimeLocale: 'gu-in'` needs to be `datetimeLocale: 'gu'`

**Review by dr3 — CHANGES_REQUESTED** [review]
The `datetimeLocale` key needs to match the moment locale for all these services :) I commented some but got bored :P All of them need correcting I think haha

File: src/app/lib/config/services/hausa.js [hunk 6]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { cyrillicAndLatin } from '@bbc/gel-foundations/scripts';
 import { hausa as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/GMT';
+import '@bbc/psammead-locales/moment/ha';
```
Comment (DenisHdz) [inline src/app/lib/config/services/hausa.js:5]: `datetimeLocale: 'ha-gh'` needs to be `datetimeLocale: 'ha'`

File: src/app/lib/config/services/hindi.js [hunk 7]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { devanagariAndGurmukhi } from '@bbc/gel-foundations/scripts';
 import { hindi as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/Asia/Kolkata';
+import '@bbc/psammead-locales/moment/hi';
```
Comment (DenisHdz) [inline src/app/lib/config/services/hindi.js:5]: `datetimeLocale: 'hi-in'` needs to be `datetimeLocale: 'hi'`

File: src/app/lib/config/services/marathi.js [hunk 8]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { devanagariAndGurmukhi } from '@bbc/gel-foundations/scripts';
 import { marathi as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/Asia/Kolkata';
+import '@bbc/psammead-locales/moment/mr';
```
Comment (DenisHdz) [inline src/app/lib/config/services/marathi.js:5]: `datetimeLocale: 'mr-in'` needs to be `datetimeLocale: 'mr'`

File: src/app/lib/config/services/mundo.js [hunk 9]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { latinDiacritics } from '@bbc/gel-foundations/scripts';
 import { mundo as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/GMT';
+import '@bbc/psammead-locales/moment/es';
```
Comment (DenisHdz) [inline src/app/lib/config/services/mundo.js:5]: `datetimeLocale: 'es-005'` needs to be `datetimeLocale: 'es'`

File: src/app/lib/config/services/nepali.js [hunk 10]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { devanagariAndGurmukhi } from '@bbc/gel-foundations/scripts';
 import { nepali as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/Asia/Kathmandu';
+import '@bbc/psammead-locales/moment/ne';
```
Comment (DenisHdz) [inline src/app/lib/config/services/nepali.js:5]: `datetimeLocale: 'ne-np'` needs to be `datetimeLocale: 'ne'`

File: src/app/lib/config/services/russian.js [hunk 11]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { cyrillicAndLatin } from '@bbc/gel-foundations/scripts';
 import { russian as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/GMT';
+import '@bbc/psammead-locales/moment/ru';
```
Comment (DenisHdz) [inline src/app/lib/config/services/russian.js:5]: `datetimeLocale: 'ru-ru'` needs to be datetimeLocale: `'ru'`

File: src/app/lib/config/services/sinhala.js [hunk 12]
```diff
@@ -6,6 +6,7 @@ import {
   F_ISKOOLA_POTA_BBC_REGULAR,
 } from '@bbc/psammead-styles/fonts';
 import '@bbc/moment-timezone-include/tz/GMT';
+import '@bbc/psammead-locales/moment/si';
```
Comment (DenisHdz) [inline src/app/lib/config/services/sinhala.js:9]: `datetimeLocale: 'si-lk'` needs to be datetimeLocale: `'si'`

File: src/app/lib/config/services/somali.js [hunk 13]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { cyrillicAndLatin } from '@bbc/gel-foundations/scripts';
 import { somali as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/Africa/Mogadishu';
+import '@bbc/psammead-locales/moment/so';
```
Comment (DenisHdz) [inline src/app/lib/config/services/somali.js:5]: `datetimeLocale: 'so-so'` needs to be datetimeLocale: `'so'`

File: src/app/lib/config/services/tamil.js [hunk 14]
```diff
@@ -3,6 +3,7 @@ import { tamil } from '@bbc/gel-foundations/scripts';
 import { tamil as brandSVG } from '@bbc/psammead-assets/svgs';
 import { F_LATHA_BOLD, F_LATHA_REGULAR } from '@bbc/psammead-styles/fonts';
 import '@bbc/moment-timezone-include/tz/GMT';
+import '@bbc/psammead-locales/moment/ta';
```
Comment (DenisHdz) [inline src/app/lib/config/services/tamil.js:6]: `datetimeLocale: 'ta-in'` needs to be datetimeLocale: `'ta'`

**Review by DenisHdz — CHANGES_REQUESTED** [review]
Following @dr3's feedback:

File: src/app/lib/config/services/serbian.js (hunk 1 above)
Comment (AlistairGempf) [reply src/app/lib/config/services/serbian.js:9]: Is this the right thing to have done here? Use `datetimeLocale` of `sr` (latin characters) for the `lat` service and `sr-cyrl` for `cyr` service.

**Review by ryanmccombe — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by j-pendlebury — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
Import the azeri locale from 'psammead-locales`

**Review by dr3 — APPROVED** [review]
LGTM, great work :)

**Review by ghost — APPROVED** [review]

**Comment by AlistairGempf** [comment]
I've raised [#2245](https://github.com/bbc/psammead/issues/2245) to solve the issues found during testing.

**Comment by AlistairGempf** [comment]
just waiting for all the E2E stuff to be fixed.

**Pull request merged**